src/universal/decorators/requireAuth/requireAuth.js

```diff
@@ -3,7 +3,7 @@ import {push} from 'react-router-redux';
 import {localStorageVars} from '../../utils/clientOptions';
 
 let key;
-export default ComposedComponent => {
```

Comment:
For the react newbies HOC = Higher Order Components.

Glad you said so. Glad we're making this decision now. The idea with eslint is should make the code better, not worse, agreed here that conforming to the current ruleset makes the code much less expressive and clean.

@jrwells can you identify this rule and nuke it? You should be able to revert a number of changes that fit this pattern.